pom.xml

```diff
@@ -279,7 +279,7 @@
         <inbound.auth.oauth.version>6.5.3</inbound.auth.oauth.version>
         <commons-collections.version>3.2.0.wso2v1</commons-collections.version>
         <carbon.kernel.version>4.10.16</carbon.kernel.version>
-        <identity.framework.version>7.0.112</identity.framework.version>
+        <identity.framework.version>7.5.48</identity.framework.version>
```

Comment:
Need to update framework version once the PRs mentioned in the description are merged and released a new framework version.